### PR TITLE
Setting ports

### PR DESCRIPTION
**Pull request by olhabn**
@jlhughes @kgrgreer 
Please review

### CODE REVIEW

File: src/foam/nanos/export/GoogleApiCredentials.js [hunk 1]
```diff
@@ -42,7 +42,8 @@ foam.CLASS({
     },
     {
       class: 'Int',
-      name: 'port'
+      name: 'port',
+      visibility: 'RO'
```
Comment (kgrgreer) [inline src/foam/nanos/export/GoogleApiCredentials.js]: Why is this RO?

File: src/services [hunk 2]
```diff
@@ -1015,7 +1015,9 @@ p({
   "name":"googleApiCredentialsConfig",
   "lazy":false,
   "service": {
-    "class":"foam.nanos.export.GoogleApiCredentials"
+    "class":"foam.nanos.export.GoogleApiCredentials",
+    "port":64349,
```
Comment (kgrgreer) [inline src/services:1019]: Where does this number come from?

Comment (olhabn) [reply src/services:1019]: random number we're agreed on

File: src/foam/nanos/export/GoogleApiCredentials.js (hunk 1 above)
Comment (olhabn) [reply src/foam/nanos/export/GoogleApiCredentials.js]: I thought that's a good idea to make sure that no one will change port

Comment (kgrgreer) [reply src/foam/nanos/export/GoogleApiCredentials.js]: Do we need to even show it then? Could we make it hidden: true. Does anyone need to know this number? Could you add documentation:?

File: src/services (hunk 2 above)
Comment (kgrgreer) [reply src/services:1019]: Who is "we"? How did we agree on it? Can we document?

Comment (olhabn) [reply src/services:1019]: I talked to Joel when I tried to make changes for staging so I think we agreed to use this one

File: src/foam/nanos/export/GoogleApiCredentials.js (hunk 1 above)
Comment (olhabn) [reply src/foam/nanos/export/GoogleApiCredentials.js]: Fixed

**Pull request merged**